scripts/jenkins-yoctobuild-build.sh

```diff
@@ -398,8 +398,19 @@ if grep mender_servers <<<"$JOB_BASE_NAME"; then
     # Use release tool to query for available docker names.
     for docker in $($WORKSPACE/integration/extra/release_tool.py --list docker -a ); do (
 
-        git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
-        docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
+        if $WORKSPACE/integration/extra/release_tool.py --help | grep -e '--map-name FROM-TYPE SERVICE TO-TYPE'; then
+            git=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker git)
+            docker_url=$($WORKSPACE/integration/extra/release_tool.py --map-name docker $docker docker_url)
```

Comment:
Question: Also, it looks to me that this might be worth, either storing this path in a variable, or adding it to PATH for the duration of this script?

Reply:
Nice suggestion. I'll keep it in mind when we refactor the full script during [QA-106](https://tracker.mender.io/browse/QA-106)